pkg/cassandra/config/config.go

```diff
@@ -50,8 +50,8 @@ type Configuration struct {
 
 // Authenticator holds the authentication properties needed to connect to a Cassandra cluster
 type Authenticator struct {
-	Basic BasicAuthenticator `yaml:"basic" mapstructure:",squash"`
-	// TODO: add more auth types
```

Comment:
I don't think your change addresses the problem of "more auth types" so I would not remove the comment